release/e2e.py

```diff
@@ -2223,22 +2223,12 @@ def _check_progress(logger):
         out_dir = os.path.expanduser(GLOBAL_CONFIG["RELEASE_RESULTS_DIR"])
 
         logger.info(
-            f"Moving results dir {temp_dir} to persistent location " f"{out_dir}"
+            f"Moving results dir {temp_dir} to persistent location {out_dir}"
         )
 
         try:
-            shutil.rmtree(out_dir)
-        except Exception:
-            logger.exception(
-                f"Ran into error when clearing the destination dir: {out_dir}"
-            )
-
-        try:
-            # Use distutils.dir_util.copy_tree() instead of shutil.cptree(),
-            # which allows existing output directory.
-            from distutils.dir_util import copy_tree
-
-            copy_tree(temp_dir, out_dir)
+            # out_dir should not exist because it is cleared in run_e2e.sh
+            shutil.cptree(temp_dir, out_dir)
```

Comment:
This is only the case on buildkite instances though (that use `run_e2e.sh`) - if users directly call `e2e.py` (e.g. for local development) they can run into problems here, right?

Reply:
Ok, restored `rmtree()`

Reply:
I think you haven't pushed, yet :-)

Reply:
It should be pushed now!